deepwell-core/src/models/page.rs

```diff
@@ -77,4 +79,17 @@ impl Page {
     pub fn exists(&self) -> bool {
         self.deleted_at.is_none()
     }
+   
+    #[cfg(feature = "ftml_compat")]
+    pub fn create_pageinfo<'a, TScoring: Scoring>(&self, votes: &Votes) -> ftml::PageInfoOwned {
```

Comment:
It's borrowed, we can avoid a clone by returning a `PageInfo`. If we want to return an owned version we can make it an `into`-like method by consuming `self` to avoid cloning.

Also for naming consistency:
* `as_pageinfo` if this goes the borrow route
* `to_pageinfo` if this stays as-is (clones internal fields)
* `into_pageinfo` if this consumes `self` to produce an owned version